packages/app/src/sections/dashboard/Stake/EscrowTable.tsx

```diff
@@ -79,43 +91,98 @@ const EscrowTable = () => {
 	const openConfirmModal = useCallback(() => setConfirmModalOpen(true), [])
 	const closeConfirmModal = useCallback(() => setConfirmModalOpen(false), [])
 
+	const EscrowStatsContainer = () => (
+		<StatsContainer columnGap="25px" justifyContent="flex-end">
+			<LabelContainers columnGap="25px" justifyContent="center">
+				<LabelContainer alignItems="flex-end">
+					<Body color="secondary">{t('dashboard.stake.tabs.escrow.total')}</Body>
+					<Body color="primary">
+						{truncateNumbers(totalVestable, 4)} {t('dashboard.stake.tabs.stake-table.kwenta-token')}
+					</Body>
+				</LabelContainer>
+				<LabelContainer alignItems="flex-end">
+					<Body color="secondary">{t('dashboard.stake.tabs.escrow.fee')}</Body>
+					<Body color="primary">
+						{truncateNumbers(totalFee, 4)} {t('dashboard.stake.tabs.stake-table.kwenta-token')}
```

Comment:
we don't need a translation for this as it's just `KWENTA` in all languages.

Reply:
Yes. Fixed.